packages/vm/lib/runBlock.ts

```diff
@@ -176,19 +192,25 @@ export default async function runBlock(this: VM, opts: RunBlockOpts): Promise<Ru
     }
   }
 
+  const results: RunBlockResult = {
+    receipts: result.receipts,
+    results: result.results,
+    stateRoot: stateRoot,
+    gasUsed: result.gasUsed,
+    logsBloom: result.bloom.bitvector,
+    receiptRoot: result.receiptRoot,
+  }
```

Comment:
@jochem-brouwer This is a cool change but changes like these absolutely need to be documented. This is just too dangerous otherwise that some inconsistencies slip through or bugs are introduced, especially since this is part of the end user API.

Are we sure here this is now what we want to expose as a `RunBlockResult`, so that we can keep this stable mid-term? 🤔  I haven't actually put a lot of thought yet on this question.

Reply:
i added a changelog entry for this: https://github.com/ethereumjs/ethereumjs-vm/pull/853/commits/6d21c41072148acd3c2b9db95731bc2b5cb81648